test/ProductConstructionService.ScenarioTests/ScenarioTestBase.cs

```diff
@@ -979,10 +989,12 @@ protected static async Task WaitForMergedPullRequestAsync(string targetRepo, str
                 return;
             }
 
-            await Task.Delay(TimeSpan.FromSeconds(30));
+            await Task.Delay(WAIT_DELAY);
         }
 
-        throw new ScenarioTestException($"The created pull request for {targetRepo} targeting {targetBranch} was not merged within {attempts} minutes");
+        var waitTime = WAIT_DELAY * attempts;
```

Comment:
Wouldn't it be easier and more precise if we just started a stopwatch and outputted that?

Reply:
The delay and attempt count were already there and are quite easy to get the minute count so I just used them. Also because my guess was that we don't really need this to be precise. But can change if needed, no particular opinion on it

Reply:
Ah, never mind then